doozer

```diff
@@ -31,10 +32,29 @@ pass_runtime = click.make_pass_decorator(Runtime)
 context_settings = dict(help_option_names=['-h', '--help'])
 
 
+VERSION_QUOTE = """
+The Doozers don't mind their buildings being eaten;
+if the Fraggles didn't eat the constructions,
+the Doozers would run out of building space,
+and if they ran out of building space,
+they would have to move away from Fraggle Rock
+or else they would die.
```

Comment:
You're the best

Reply:
It was either this or an ASCII doozer face